apps/studio/src/features/dashboard/components/ResourceTable/LastEditCell.tsx

```diff
@@ -0,0 +1,37 @@
+import { Text, VStack } from "@chakra-ui/react"
+import { formatDistance } from "date-fns"
+
+import type { ResourceTableData } from "./types"
+
+export interface LastEditCellProps {
+  date: ResourceTableData["lastEditDate"]
+  email: ResourceTableData["lastEditUser"]
+}
+
+export const LastEditCell = ({
+  date,
+  email,
+}: LastEditCellProps): JSX.Element => {
+  if (date === "folder") {
+    return (
+      <Text as="span" textStyle="caption-2">
+        -
+      </Text>
+    )
+  }
```

Comment:
alternatives we can consider: 
1. showing last edited time of direct child pages
2. showing last edited time of all child pages + folder

i think here just using `-` is actually good enough but just cc @sehyunidaaa if you care. i think using `-` is actually really good for me and i hate 2 + dislike 1.